src/frontend-scripts/components/section-main/Profile.jsx

```diff
@@ -373,6 +369,24 @@ class ProfileWrapper extends React.Component {
 
 	render() {
 		const { profile } = this.props;
+		const getTimestamp = ts => {
+			const pad = (n, s = 2) => `${new Array(s).fill(0)}${n}`.slice(-s);
+			const d = new Date(ts);
+			return `${pad(d.getFullYear(), 4)}-${pad(d.getMonth() + 1)}-${pad(d.getDate())}`;
+		};
+
+		const blacklistInfo = element => {
+			if (typeof element === 'string') {
+				return <a href={`/game/#/profile/${element}`}>{element}</a>;
+			} else {
+				return (
+					<span>
+						<a href={`/game/#/profile/${element['userName']}`}>{element['userName']}</a>&nbsp;
+						{getTimestamp(element['timestamp'])} {element['reason'] ? `: ${element['reason']}` : ''}
```

Comment:
the two spaces around the colon are wonky imo

Reply:
fixed!